code/modules/hydroponics/grown.dm

```diff
@@ -89,6 +89,12 @@
 /obj/item/reagent_containers/food/snacks/grown/attack_self(mob/user)
 	if(seed && seed.get_gene(/datum/plant_gene/trait/squash))
 		squash(user)
+	if(seed.get_gene(/datum/plant_gene/trait/noreact))
+		user.visible_message("<span class='warning'>[user] shakes [src] vigorously!</span>", "<span class='userdanger'>You shake [src] vigorously!</span>")
+		log_bomber(user, "primed a", src, "for detonation")
+		if(iscarbon(user))
+			var/mob/living/carbon/C = user
+			C.throw_mode_on()

```

Comment:
Don't turn throw mode on, again sep chems doesn't always have to be treated like a grenade

Reply:
well, works exactly like a chem grenade, but also done